.github/workflows/release.yml

```
@@ -32,6 +32,18 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: 18
+      - uses: actions/checkout@v4
+      - name: Set git user
+        uses: fregante/setup-git-user@v1
+      - name: Generate Api docs
+        run: |
+          npm ci
+          npm run doc
+      - name: Push updated markdown docs
+        uses: stefanzweifel/git-auto-commit-action@v5
+        with:
+          commit_message: 'feat: update api markdown docs'
```

Nit: I think this should be `docs`, not `feat`.

```suggestion
          commit_message: 'docs: update API docs'
```